mpas_analysis/shared/mpas_xarray/mpas_xarray.py

```diff
@@ -261,6 +289,64 @@ def remove_repeated_time_index(ds):  # {{{
     return ds  # }}}
 
 
+def map_variable(variable_name, ds, variable_map):
+    """
+    Find the variable (or list of variables) in dataset ds that map to the
+    mpas_analysis variable given by variable_name.
+
+    variable_map is a dictionary with keys that are variable names used by
+    MPAS-Analysis and values that are lists of possible names for the same
+    variable in the MPAS dycore that produced the data set (which may differ
+    between versions).
+
+    Xylar Asay-Davis
+    12/04/2016
+    """
+    possible_variables = variable_map[variable_name]
+    for var in possible_variables:
+        if isinstance(var, (list, tuple)):
```

Comment:
I'm glad you are supporting lists here for greater generality.  That is really nice.